plugins/SimulationView/LayerSlider.qml

```diff
@@ -193,6 +193,12 @@ Item {
             return result
         }
 
+        function setValueAndStopSimulation(value)
```

Comment:
I will call it differently since this component doesn't know about simulation. For instance `setValueManually`